src/Components/Home.jsx

```diff
@@ -7,12 +7,18 @@ import Statement from './Statement.jsx'
 const Home = () => {
   return ( 
     <div>
-      <header>{ Header }</header>
-      <section>{ Statement }</section>
-      <section>{ Offerings }</section>
-      <footer>{ Footer }</footer>
+      <Header />
+      <Statement />
+      <Offerings />
+      <Footer />
     </div>
   )
 }
 
```

Comment:
This makes sense too. I got turned around with navigating the architecture with the Vite template that I added these semantic elements to the HTML when it should have been just the components